superset/assets/src/SqlLab/components/SqlEditor.jsx

```diff
@@ -453,25 +458,36 @@ class SqlEditor extends React.PureComponent {
       </div>
     );
   }
+  leftBar() {
+    return (
+      <CSSTransition
+        classNames="schemaPane"
+        in={!this.props.hideLeftBar}
+        timeout={300}
+      >
+        <SqlEditorLeftBar
+          database={this.props.database}
+          queryEditor={this.props.queryEditor}
+          tables={this.props.tables}
+          actions={this.props.actions}
+        />
+      </CSSTransition>
+    );
+  }
   render() {
     return (
-      <div ref={this.sqlEditorRef} className="SqlEditor">
-        <CSSTransition
-          classNames="schemaPane"
-          in={!this.props.hideLeftBar}
-          timeout={300}
-        >
-          <div className="schemaPane">
-            <SqlEditorLeftBar
-              database={this.props.database}
-              queryEditor={this.props.queryEditor}
-              tables={this.props.tables}
-              actions={this.props.actions}
-            />
-          </div>
-        </CSSTransition>
+      <Split
+        ref={this.sqlEditorRef}
+        className={this.props.hideLeftBar ? 'SqlEditor-expanded' : 'SqlEditor'}
+        sizes={[INITIAL_WEST_PERCENT, INITIAL_EAST_PERCENT]}
```

Comment:
do you want user's setting be persist when they reload or switch browser?

Reply:
I don't think this should be persisted, since in my experience it's more circumstantial. For some of our tables the metadata is too big and users want momentarily to expand it. What are your thoughts?